README.md

```diff
@@ -477,6 +477,24 @@ More information on why this feature was needed can be found in [the related tas
     yarn expand-wildcard-domains platforms scripts/wildcard-domain-processor/wildcard_domains.json
     ```
 
+1. **How wildcard expansion works**
+
+    The wildcard expansion process uses the `expandWildcardsInAst` function to process rules
+    in the Abstract Syntax Tree (AST) format. The function handles different rule categories:
+
+    - **Network Rules** — all network rules with wildcard domains are expanded.
+
+    - **Cosmetic Rules** — only element hiding rules and their exceptions are expanded
+      since these are natively supported in Safari content blockers.
+      Other cosmetic rules (like scriptlets) are not expanded
+      because wildcards are not problematic and do not need expansion since they are applied by:
```

Comment:
unclear wording - handled by `Advanced Blocking` and `tswebextension` under the hood?

Reply:
yes. everything is already there
![slaKnJfLthkLAvyusD](https://github.com/user-attachments/assets/8fd88d29-3d9b-4d59-bc6c-4859c300b689)